tests/wamr-test-suites/wasi-test-script/run_wasi_tests.sh

```diff
@@ -41,6 +41,12 @@ readonly THREAD_INTERNAL_TESTS="${WAMR_DIR}/core/iwasm/libraries/lib-wasi-thread
 readonly THREAD_STRESS_TESTS="${WAMR_DIR}/core/iwasm/libraries/lib-wasi-threads/stress-test/"
 readonly LIB_SOCKET_TESTS="${WAMR_DIR}/core/iwasm/libraries/lib-socket/test/"
 
+add_env_key_to_test_config_file() {
+    filepath="tests/$2/testsuite/$3.json"
```

Comment:
Instead of modifying test files, could we perhaps extend the WAMR adapter and allow passing env variables through it?

Reply:
Opened a PR, https://github.com/WebAssembly/wasi-testsuite/pull/99.